targets/lenovo-x1/audiovmExtraModules.nix

```diff
@@ -0,0 +1,91 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+{
+  lib,
+  pkgs,
+  microvm,
+  configH,
+  ...
+}: let
+  # TCP port used by Pipewire-pulseaudio service
+  pulseaudioTcpPort = 4713;
```

Comment:
I'm wondering how much of this stuff is actually lenovo-x1 specific?

Reply:
Later when we might adopt audiovm to other targets, most of this stuff is probably going to be the same, so at least later this should be refactored to common microvm audovm.nix file